backend/routes/performance_routes.py

```diff
@@ -1,8 +1,46 @@
 from flask import Blueprint
-from flask import jsonify
+from flask import jsonify, request
+import simplejson as json
+from ..db.models import Performance
 
 blueprint = Blueprint('performance', __name__, url_prefix='/performances')
 
 @blueprint.route('/')
 def main():
-	return 'Welcome to Dancefest Backend!'
\ No newline at end of file
+	return 'Welcome to Dancefest Performances Backend!'
+
+@blueprint.route('/events/<event_id>/performances/<performance_id>', methods=['POST'])
+def edit_performance_details(event_id, performance_id):
+	'''
+	requires post body in the following format:
+	{
+		school: <string>,
+		performers: Array<string>,
+		dance_title: <Int>,
+		dance_style: <string>,
+		dance_entry: <Int>,
+		dance_size: <String>,
+		competition_level: <String>,
+		choreographers: Array<string>,
+		academic_level: <string>	
+	}
+	Returns data in the following format
+	{
+		school: <string>,
+		performers: Array<string>,
+		dance_title: <Int>,
+		dance_style: <string>,
+		dance_entry: <Int>,
+		dance_size: <String>,
+		competition_level: <String>,
+		choreographers: Array<string>,
+		academic_level: <string>	
+	}
+	'''
+	performance = Performance.query.get(performance_id)
+	performance_json = request.get_json()
+	performance.update(**performance_json)
```

Comment:
this won't update the `performance` object. replace with `performance = performance.update(**performance_json)`